Formula/telnet.rb

```diff
@@ -0,0 +1,44 @@
+class Telnet < Formula
+  desc "Apple telnet built from macOS 10.12.4 sources"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/remote_cmds/remote_cmds-54.50.1.tar.gz"
+  sha256 "156ddec946c81af1cbbad5cc6e601135245f7300d134a239cda45ff5efd75930"
+
+  keg_only :provided_pre_high_sierra
+  depends_on :xcode => :build
```

Comment:
Should be a blank line between the `keg_only` and `depends_on`. Very minor nit, but.